src/test/infrastructure/bootstrap/TcfApiInitializerTest.js

```diff
@@ -2,10 +2,20 @@ import 'jsdom-global/register'
 import {expect} from 'chai'
 import {TcfApiInitializer} from '../../../main/infrastructure/bootstrap/TcfApiInitializer'
 import {waitCondition} from '../../../main/core/service/waitCondition'
+import {BorosTcf} from '../../../main/application/BorosTcf'
 
 describe('TcfApiInitializer', () => {
-  it('should return the registered the window.__tcfapi command consumer', () => {
-    const tcfapi = TcfApiInitializer.init({window})
+  it('should return instance of BorosTcf with the methods', () => {
+    const borosTcf = TcfApiInitializer.init({window})
+    expect(borosTcf instanceof BorosTcf).to.be.true
+    expect(typeof borosTcf.getVendorList === 'function').to.be.true
+    expect(typeof borosTcf.saveUserConsent === 'function').to.be.true
+    expect(typeof borosTcf.loadUserConsent === 'function').to.be.true
+    expect(typeof borosTcf.saveUserConsent === 'function').to.be.true
+  })
+  it('should register the window.__tcfapi command consumer', () => {
+    TcfApiInitializer.init({window})
+    const tcfapi = window.__tcfapi
     expect(tcfapi).to.be.a('function')
     expect(tcfapi).to.equal(window.__tcfapi)
```

Comment:
this expect seems a little bit redundant

Reply:
I agree. Removed. Thanks.